endpoints/auction_test.go

```diff
@@ -397,11 +397,22 @@ func TestShouldUsersync(t *testing.T) {
 			t.Errorf("Expected syncs: %t, allowed syncs: %t", expectAllow, allowSyncs)
 		}
 	}
-	doTest("0", "", false, false, true)
-	doTest("1", "", true, true, false)
-	doTest("1", "a", true, false, false)
-	doTest("1", "a", false, true, false)
-	doTest("1", "a", true, true, true)
+
+	doTest("0", "", false, false, false)
```

Comment:
If you're willing to convert this to a table driven test, I find this:
```
 {signal: "0", consent: "", allowBidderSync: true, allowHostCookies: false, expectAllow: true 
```
much easier to read.